common/script/content/eggs.js

```diff
@@ -0,0 +1,262 @@
+import {
+  assign,
+  defaults,
+  each,
+} from  'lodash';
+import t from './translation';
+
+function applyEggDefaults (set, config) {
+  each(set, (egg, key) => {
+    defaults(egg, {
+      canBuy: config.canBuy,
+      value: 3,
+      key,
+      notes: t('eggNotes', {
+        eggText: egg.text,
+        eggAdjective: egg.adjective,
+      }),
+      mountText: egg.text,
+    });
+  });
+}
+
+function hasQuestAchievementFunction (key) {
+  return (user) => {
+    return user.achievements.quests &&
+      user.achievements.quests[key] > 0;
+  };
+}
+
+let drops = {
+  Wolf: {
```

Comment:
The problem is the way the content api works. Let's say we decided to drop the `Cub` convention, but maintain the keys for the ones that had them before, then you'd then have this:

```js
let eggs = Object.keys(content.eggs);
eggs; // ['Wolf', 'TigerCub', 'Tiger', 'PandaCub', 'Panda', 'LionCub', 'Lion', 'Fox', 'FlyingPig', 'Dragon', 'Cactus', 'BearCub', 'Bear']
```

So if you were to loop over the eggs to display the different kind of drop eggs available, you'd suddenly list duplicates of a few.